lms/djangoapps/verify_student/management/commands/approve_id_verifications.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django admin commands related to verify_student
+"""
+
+
+import logging
+import os
+from pprint import pformat
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.core.management.base import BaseCommand, CommandError
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
+from lms.djangoapps.verify_student.utils import earliest_allowed_verification_date
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    This command manually approves ID verification attempts for a provided set of learners whose ID verification
+    attempt is in the submitted state.
+
+    This command differs from the similar manual_verifications command in that it approves the
+    SoftwareSecurePhotoVerification instance instead of creating a ManualVerification instance. This is advantageous
+    because it ensures that the approval is registered with the Name Affirmation application to approve corresponding
+    verified names. Creating a ManualVerification instance does not effect any change in the Name Affirmation
+    application.
+    
+    Example usage:
+        $ ./manage.py lms idv_verifications <absolute path of file with user IDs (one per line)>
+    """
+    help = 'Manually approves ID verifications for users with an ID verification attempt in the submitted state.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
```

Comment:
I changed this to take a file of user IDs instead of emails to limit PII exposure.